feature/chat/src/main/java/com/serranoie/app/feature/chat/ChatViewModel.kt

```
@@ -0,0 +1,293 @@
+        return try {
+            val time = timestamp.toLongOrNull()
+            if (time != null) {
+                val date = java.util.Date(time)
+                java.text.SimpleDateFormat("h:mm a", java.util.Locale.getDefault()).format(date)
+            } else {
+                try {
+                    val isoFormatter = java.time.format.DateTimeFormatter.ISO_LOCAL_DATE_TIME
+                    val localDateTime = java.time.LocalDateTime.parse(timestamp, isoFormatter)
+                    val timeFormatter = java.time.format.DateTimeFormatter.ofPattern("h:mm a")
+                    localDateTime.format(timeFormatter)
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)